mods/cnc/maps/nod02a/map.yaml

```diff
@@ -0,0 +1,414 @@
+MapFormat: 7
+
+RequiresMod: cnc
+
+Title: Invasion of Egypt (2a)
```

Comment:
Can you leave the `2` out, so it matches the other missions?  At one point I'd like have all mission titles start with `GDIxy:` or `Nodxy:`, like we do in RA, but let's got for conformity while that hasn't been done yet.